tests/e2e/specs/modules/adsense/amp-auto-ads-tag.test.js

```diff
@@ -45,11 +46,16 @@ describe( 'AMP <amp-auto-ads> tag', () => {
 
 	it( 'is output in primary mode', async () => {
 		await setAMPMode( 'primary' );
-		await expect( '/hello-world' ).toHaveAMPAutoAdsTag();
+		await expect( createURL( '/hello-world' ) ).toHaveAMPAutoAdsTag();
 	} );
 
 	it( 'is output in secondary mode', async () => {
 		await setAMPMode( 'secondary' );
-		await expect( '/hello-world' ).toHaveAMPAutoAdsTag();
+		await expect( createURL( '/hello-world', 'amp' ) ).toHaveAMPAutoAdsTag();
```

Comment:
Originally I was thinking the query could be passed through in the first argument to `createURL` but it gets URL encoded, so this is fine since there isn't a great way to pass the query separately. It's possible to pass through as an argument to the matcher itself, but `toHaveAMPAutoAdsTag( 'amp' )` would be weird.